demisto_sdk/commands/json_to_outputs/json_to_outputs.py

```diff
@@ -198,7 +205,7 @@ def parse_json(data, command_name, prefix, verbose=False, interactive=False):
     return yaml_output
 
 
-def json_to_outputs(command, input, prefix, output=None, verbose=False, interactive=False):
+def json_to_outputs(command, input, prefix, output=None, verbose=False, interactive=False, descriptions=None):
```

Comment:
shouldn't this be called `description_json` for kwargs to work?

Reply:
It's intentionally called `descriptions` here, as it doesn't _have_ to be a JSON here (but can be some string used as a flag*, to trigger prompting the user to input the JSON as `multiline_input()`.

We definitely can call it `description_json` for better kwarg compatibility, should we?

Reply:
renamed all to `descriptions`.